src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
is this file needed?

Reply:
According to https://www.tutorialspoint.com/spring_boot/spring_boot_application_properties.htm
this file contains spring's variables, which we're not using at the moment.
Rather than removing this, I'm tempted to add `server.port` and `spring.application.name` to make its purpose more obvious,

Reply:
Some of the things you might put in here soon are the database connection details, or any variables you want to set differently between Test, Dev and prod :)